src/EPPlus/FormulaParsing/Excel/Functions/Math/CeilingMath.cs

```diff
@@ -31,8 +31,8 @@ public override CompileResult Execute(IEnumerable<FunctionArgument> arguments, P
             ValidateArguments(arguments, 1);
             if (arguments.ElementAt(0).Value == null) return CreateResult(0d, DataType.Decimal);
             var number = ArgToDecimal(arguments, 0, context.Configuration.PrecisionAndRoundingStrategy);
-            var significance = (arguments.Count() > 1) ? ArgToDecimal(arguments, 1) : 1;
-            var mode = (arguments.Count() > 2) ? ArgToDecimal(arguments, 2) : 0d;
+            var significance = (arguments.Count() > 1) ? ArgToDecimal(arguments, 1, 1D) : 1;
+            var mode = (arguments.Count() > 2) ? ArgToDecimal(arguments, 2, 0D) : 0;
```

Comment:
Shouldn't the 0 option after : still be 0d? Hardly matters but could be good to specify that its a decimal still. Unless C# handles that?